src/js/components/agencyV2/visualizations/totalObligationsOverTime/TotalObligationsOverTimeVisualization.jsx

```diff
@@ -137,63 +140,98 @@ const TotalObligationsOverTimeVisualization = ({
         }
     }, [xDomain, todaysDate]);
 
+    const toggleTooltipVisibility = (isVisible) => {
+        setTooltipIsVisible(isVisible);
+    };
+
+    const balance = agencyBudget - data[data.length - 1].obligated;
+    const percentOfTotal = calculatePercentage(agencyBudget - data[data.length - 1].obligated, agencyBudget);
```

Comment:
Is it guaranteed that the final obligation value in the array is always the largest?

Reply:
No, it's not necessarily largest obligation but its the last reported obligation (which it should be).  This array is sorted by period. 
![Screen Shot 2022-01-05 at 11 40 47 AM](https://user-images.githubusercontent.com/121010/148255544-fb953bb8-d7bd-46fd-84d7-73fcd49606dd.png)